File: app/src/main/java/com/inqint/yarnrequirements/Projects/Blanket.java
package com.inqint.yarnrequirements.Projects;

/**
 * Created by deb on 4/27/16.
 */
public class Blanket extends Scarf {

    public Blanket(String name, int thumbImageID, int imageID, Class<?> fragmentClass) {
        super(name, thumbImageID, imageID, fragmentClass);
    }

    @Override
    public void calcYarnRequired() {
        super.calcYarnRequired();

        yarnNeeded = (int)(1.2 * (double)yarnNeeded); // Add 20% to estimate
        calcBallsNeeded();
    }
}

File: app/src/main/java/com/inqint/yarnrequirements/Projects/Blanket.kt
package com.inqint.yarnrequirements.Projects

/**
 * Created by deb on 4/27/16.
 */
class Blanket(name: String, nameID: Int, thumbImageID: Int, imageID: Int) :
    Scarf(name, nameID, thumbImageID, imageID) {

    override fun calcYarnRequired() {
        super.calcYarnRequired()

        yarnNeeded = (1.2 * yarnNeeded.toDouble()).toInt() // Add 20% to estimate
        calcBallsNeeded()
    }
}
